addOns/commonlib/commonlib.gradle.kts

```diff
@@ -0,0 +1,128 @@
+import org.zaproxy.gradle.addon.AddOnStatus
+
+plugins {
+    `maven-publish`
+    signing
+}
+
+group = "org.zaproxy.addon"
+
+version = "0.1.0"
```

Comment:
Start with 1.0.0 already?

Reply:
I think it'd be fine to start with 1.0.0